src/common/style/vanilla/css/breadcrumbs.css

```diff
@@ -0,0 +1,25 @@
+.p-breadcrumbs {
```

Comment:
Currently I decided to commit the compiled CSS modules.

In theory, we could not version control them and just build them when we run server or build for production, but I assume it will be easier to keep them in the code as they will not change much (only if we update vanilla version) and it would save us getting more complex build tasks to make sure these are properly build with rest of the webpack stuff...